src/config.ts

```diff
@@ -0,0 +1,20 @@
+import * as env from 'env-var';
+
+export default {
+  server: {
+    port: env.get('PORT').default('8000').asPortNumber().toString(),
+  },
+  services: {
+    user: {
+      url: env.get('USER_SERVICE_URL').required().asUrlString(),
+      resourceName: 'users',
+    },
+    group: {
+      url: env.get('GROUP_SERVICE_URL').required().asUrlString(),
+      resourceName: 'groups',
+    },
+    userServiceUrl: env.get('USER_SERVICE_URL').required().asUrlString(),
+    groupServiceUrl: env.get('GROUP_SERVICE_URL').required().asUrlString(),
+  },
+  userHeader: env.get('USER_HEADER').default('X-User-ID').asString().toLowerCase(),
+};
```

Comment:
attach default values to the urls

Reply:
I do not agree with you. I want the `env-var` to throw an exception when a URL for one of the services is not given.